libs/components/src/lib/click-outside/click-outside.directive.ts

```diff
@@ -17,7 +7,6 @@ import { Subscription } from 'rxjs'
  */
 @Directive({ selector: '[scClickOutside]', standalone: true })
 export class ClickOutsideDirective implements OnDestroy, OnChanges {
-  // tslint:disable-next-line:no-input-rename
```

Comment:
why is `eslint` not complaining about that, compared to `tslint`?

Reply:
Since we don't change the name, that existing input name if I change it to something else we get an error 
<img width="902" alt="Screenshot 2024-12-12 at 15 28 23" src="https://github.com/user-attachments/assets/8fbbf912-cc1b-43b3-983b-46ddb108620b" />